### ISSUE TITLE
Typo in SQL keyword: ESCEPTION

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#560]
(Original issue 853 created by roskakori on 2013-02-21T14:56:06.023897+00:00)

There seems to be a typo in ``pygments.lexers.sql.SqlLexer``:

```
#!python
r'EXCEPT|ESCEPTION|EXCLUDING|EXCLUSIVE|EXEC|EXECUTE|EXISTING|'
```

Presumably ``ESCEPTION`` should read ``EXCEPTION``.

**Comment by Anteru** [comment]
(Original issue 853 last updated on 2013-05-19T09:48:47.197346+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2013-05-19T09:48:47.190948+00:00)

Closes #853: fix typo in SQL lexer.

→ <<cset 1087ccc1af73>>
